apps/frontend/src/modules/users/users-api.ts

```diff
@@ -21,15 +22,23 @@ class UserApi extends BaseHTTPApi {
 		super({ baseUrl, http, path: APIPath.USERS, storage });
 	}
 
-	public async getAll(): Promise<UserGetAllResponseDto> {
-		const response = await this.load(
-			this.getFullEndpoint(UsersApiPath.ROOT, {}),
+	public async getAll({
+		page,
+		pageSize,
+	}: PaginationQueryParameters): Promise<UserGetAllResponseDto> {
+		const end = this.getFullEndpoint(
+			UsersApiPath.ROOT,
+			"?page=:page&pageSize=:pageSize",
```

Comment:
getFullEndpoint is for endpoint only (without query params). Pass query to load options. And something like private getUrl method to base http api and use `URLSearchParams` to build query string in it 
```suggestion
	public async getAll(query: PaginationQueryParameters): Promise<UserGetAllResponseDto> {
    const response = await this.load(
			this.getFullEndpoint(UsersApiPath.ROOT, {}),
			{
				contentType: ContentType.JSON,
				hasAuth: true,
				method: "GET",
				query
			},
```

Reply:
Done it. Thanks!